### PR TITLE
Status Codes user guide

### PR DESCRIPTION
**Pull request by larry-safran**

### CODE REVIEW

**Comment by larry-safran** [comment]
@temawi Needs a review and approval

File: content/en/docs/guides/status-codes.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+for a given RPC. Applications should only use values defined below. 
+
+The gRPC client and server-side implementations may generate and
+return status on their own when errors happen. Only a subset of
+the pre-defined status codes are generated by the gRPC libraries. This
+allows applications to be sure that any other code it sees was actually
+returned by the application (although it is also possible for the
+server-side to return one of the codes generated by the gRPC libraries).
+
+See the [Error handling](https://github.com/grpc/grpc.io/blob/main/content/en/docs/guides/error.md) user guide
```
Comment (ejona86) [inline content/en/docs/guides/status-codes.md:19]: This should link to the user guide.

**Review by ejona86 — APPROVED** [review]

File: assets/scss/_grpc.scss [hunk 2]
```diff
@@ -119,6 +119,10 @@ sections site wide to the original site's background svg.
   max-width: 1140px;
 }
 
+.td-content table {
```
Comment (temawi) [inline assets/scss/_grpc.scss:122]: Add a comment as to why this is needed.

File: content/en/docs/guides/status-codes.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+     Explains the status codes used in gRPC.
+---
+
+### Overview
+All RPCs started at a client return a `status` object composed of an integer
+code and a string error description. The server-side chooses the status it returns
+for a given RPC. Applications should only use values defined below. 
+
+The gRPC client and server-side implementations may generate and
+return status on their own when errors happen. Only a subset of
```
Comment (temawi) [inline content/en/docs/guides/status-codes.md:13]: Is this clearer?

"When an error situation occurs, the gRPC library may produce a corresponding status code. The library may do this either on the client- or the server-side."

Comment (larry-safran) [reply content/en/docs/guides/status-codes.md]: Incorporated with "status code" replaced by "`status`"

File: content/en/docs/guides/status-codes.md (hunk 1 above)
Comment (larry-safran) [reply content/en/docs/guides/status-codes.md]: fixed

File: assets/scss/_grpc.scss (hunk 2 above)
Comment (larry-safran) [reply assets/scss/_grpc.scss:123]: done

**Pull request merged**